src/README.rst

```diff
@@ -19,6 +19,7 @@ Change Log
 Unreleased
 -----
 - Add application health policies parameter for config upgrade, which are available in Service Fabric runtime 6.3 (#92)
+- Add command to see the current version and target azure-servicefabric version (#26)
```

Comment:
You want to reference the PR number not the issue number, so this should be `#104`